test/BadgeSpec.tsx

```diff
@@ -0,0 +1,60 @@
+import { render } from '@testing-library/react';
+
+import Badge from '../src/Badge';
+
+describe('Badge', () => {
+  it('Should render correctly', () => {
+    const { getByTestId } = render(
+      <Badge bg="primary" pill data-testid="test">
+        Message
+      </Badge>,
+    );
+
+    const badge = getByTestId('test');
+    badge.innerText.should.contains('Message');
```

Comment:
```suggestion
    badge.innerText.should.equal('Message');
```
would be a tad more specific